.gn

```diff
@@ -166,3 +166,9 @@ exec_script_whitelist =
       "//tools/grit/grit_rule.gni",
       "//tools/gritsettings/BUILD.gn",
     ]
+
+exec_script_whitelist += [
+  "//starboard/tools/BUILD.gn",
+  "//starboard/build/config/sabi/BUILD.gn",
+  "//starboard/build/platform_path.gni"
+]
```

Comment:
Tiny nit: No EOL.

Note that pre-commits haven't been re-instated yet, so manual checks are needed for now.

Reply:
this file is upstream , so no need to change it ?